### PR TITLE
minor capitalisation fix

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: mscore/inspector/inspectorBase.cpp [hunk 1]
```diff
@@ -531,7 +531,7 @@ void InspectorBase::mapSignals(const std::vector<InspectorItem>& il, const std::
                         QMenu* menu = new QMenu(this);
                         resetButton->setMenu(menu);
                         resetButton->setPopupMode(QToolButton::MenuButtonPopup);
-                        QAction* a = menu->addAction(tr("set style"));
+                        QAction* a = menu->addAction(tr("Set style"));
```
Comment (Jojo-Schmitz) [inline mscore/inspector/inspectorBase.cpp]: Maybe better "Set as style" or "Use as style setting"?

Comment (IsaacWeiss) [inline mscore/inspector/inspectorBase.cpp]: I like "Set as style."

Comment (Jojo-Schmitz) [inline mscore/inspector/inspectorBase.cpp]: If that indeed is what that button is all about. And if it it, it doesn't really work, see https://musescore.org/en/node/151491#comment-619726

**Comment by lasconic** [comment]
I take this one on me.

**Pull request closed without merging**